ckanext/ogdchcommands/commands.py

```diff
@@ -39,9 +41,14 @@ class OgdchCommands(ckan.lib.cli.CkanCommand):
         # - the results are written in the shaclresults directory
         # - both of these directories are specified in the ckan
         #   configuration file
-
         paster ogdch shacl_validate
             {source_id} --shapefile={name of the shape file}
+
+        # Publish scheduled datasets
+        # checks for private datasets that have a scheduled date
+        # that is either today or in the past and sets them to public
+        paster ogdch publish_scheduled_datasets [--dryrun}]
```

Comment:
I think you don't need the `}` in this line, and it should be removed from line 33 as well.